coffeecard/CoffeeCard/Models/User.cs

```diff
@@ -30,8 +36,12 @@ public User()
         public virtual ICollection<Ticket> Tickets { get; set; }
         public virtual ICollection<Purchase> Purchases { get; set; }
         public virtual ICollection<Statistic> Statistics { get; set; }
+        [Required]
         public bool IsVerified { get; set; }
+        [Required]
         public bool PrivacyActivated { get; set; }
+        [Required]
+        public bool IsBarista { get; set; }
```

Comment:
Same as above, we may want to make it an enum, considering there is more than just baristas